test/e2e/ComponentGlobal.test.js

```diff
@@ -0,0 +1,32 @@
+const { execute } = require("../../lib");
+const { createMockStreams, resourceFile, allArgs } = require("./E2ETests");
+const lolex = require("lolex");
+
+describe("m.global usage in scenegraph components", () => {
+    let outputStreams;
+    let clock;
+
+    beforeAll(() => {
+        clock = lolex.install({ now: 1547072370937 });
```

Comment:
I don't think we're doing anything in these tests that requires mocking out the current system time, so this (and all other references to `clock` in this file) can be removed 😃